client/py/aou_workbench_client/config.py

```diff
@@ -0,0 +1,22 @@
+"""Reads configuration settings from disk.
+"""
+
+import json
+
+# Constants for keys in the configuration
+ALL_OF_US_CONFIG_FILE = 'all_of_us_config.json'
+WORKSPACE_NAMESPACE_KEY = 'WORKSPACE_NAMESPACE'
+WORKSPACE_ID_KEY = 'WORKSPACE_ID'
+API_HOST_KEY = 'API_HOST'
+BUCKET_NAME_KEY = 'BUCKET_NAME'
+
+class AllOfUsConfig(object):
+    def __init__(self):
+        with open(ALL_OF_US_CONFIG_FILE) as config_file:
```

Comment:
Does this imply that the current working directory of the python process will be the same directory where config files are localized?

Reply:
Yes, at the moment that's how things are. We could create a directory for config stuff instead... let me know what you think. (I'd address it in a follow-up.)

Reply:
Yes, I think it will be helpful to have a specific directory for config. Based on your proposal for the way we lay out directories in the bucket itself, we might just want to mirror that proposal directly. The main thing is that when researchers view the notebook root directory in the Jupiter browser, I don't want them to be confused by seeing their own notebooks intermingled with files they didn't author and may not understand. For this configuration file in particular there may even be an argument for localizing it to a path that is outside of the Jupiter notebook route (if that's an option) since we don't expect researchers to have a reason to view and edit this file directly.